src/Equinor.ProCoSys.BusSender.Worker/Program.cs

```
@@ -143,9 +147,27 @@ private static IHostBuilder CreateHostBuilder(string[] args)
     public static async Task Main(string[] args)
     {
         using var host = CreateHostBuilder(args).Build();
-        ILogger? logger = host.Services.GetService<ILogger<Program>>();
+        using var scope = host.Services.CreateScope();
+        var plantService = host.Services.GetService<IPlantService>();
+        var plantRepository = scope.ServiceProvider.GetService<IPlantRepository>();
+        var plants = new List<string>();
+
+        if (plantRepository != null)

```

If plantsRepository is null, then plants will be an empty list.

Is this serious enough to throw an exception as "fail fast" and notify us quickly that something is wrong?
What are the consequences of an empty plants list, will all messages be processed or no messages processed or something in between?

---

In the case where no plants are provided resulting in empty list, no messages will be processed. Maybe this is an undesired state which should lead to an exception and web job not starting. Not sure. Could be confusing to have an instance running in this state. E.g. the risk of not discover when config has not been read properly.

---

Implemented fail fast solution.